common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -597,16 +618,19 @@ def get_transcript(self, transcript_format='srt', lang=None):
 
         return content, filename, Transcript.mime_types[transcript_format]
 
-    def get_default_transcript_language(self):
+    def get_default_transcript_language(self, transcripts, bumper=False):
```

Comment:
> Nit: It seems both the transcripts and bumper parameters specify the same information. Can we just do with bumper so that callers do not have to worry about ensuring both are in sync?

Fixed, changed interface.